ADA/ada.qmd

```diff
@@ -19,57 +17,234 @@ It brings together:
 ---
 
-## Walkthrough a simple analysis
+### Working with data and code
 
-See the [ADA support page on using R, python and SQL in Databricks](https://analytical-data-access.azurewebsites.net/get-support/using-r-python-and-sql-in-databricks)
+---
 
+#### Getting access to your data
 
 ---
 
-## Helpful getting started videos
+Data in Databricks is held in catalogs in the Delta Lake, which is centrally governed by the ADA team. Catalogs provide different levels of access control and security. In Databricks, databases are referred to as "schemas", with tables sitting below them. You can find out more information about this on the [ADA guidance site](https://analytical-data-access.azurewebsites.net/get-support/how-databricks-works). 
 
-See the collection of [getting started videos](https://educationgovuk.sharepoint.com/:f:/r/sites/ADA-SuperUserForum/Shared%20Documents/General/Getting%20started%20videos?csf=1&web=1&e=RCFSib) covering the basics of Databricks and using notebooks. 
+You will need to be given access to Databricks and the data you require by the ADA team by completing an "Analytical Data Access Request" service request form in the IT Help Centre. At the moment, not all of the data currently available in SSMS has been migrated to the Delta Lake for use in Databricks. You can [check the status of various datasets](https://app.powerbi.com/groups/me/reports/966389db-2e4e-42b6-8416-36aee2fb9503/ReportSection?ctid=fad277c9-c60a-4da1-b5f3-b3b8b34a82f9&experience=power-bi) in the data catalogue dashboard. 
+
+To access your data and make use of cloud computing resource, you will need access to a cluster, which is a group of virtual machines that complete your processing for you. Your cluster setup will depend on which programming language you wish to use. You will be given access to a shared cluster when your data access is granted. Please see the [how clusters work](https://analytical-data-access.education.gov.uk/get-support/how-clusters-work) page on the ADA guidance site for more detail. 
+
+---
+
+#### Writing and running code: notebooks
+
+---
+
+In the Databricks interface, you have the choice to use notebooks, scripts or the Databricks SQL Editor. 
+
+Notebooks are easy to use in Databricks, and might be something you've not come across before. Notebooks are made up of multiple code or markdown cells, which may mean that you have to structure your code in a different way than you're used to. They are different to IDEs (Integrated Development Environments) like RStudio. You can find more information on Databricks notebooks and their structure on the [ADA guidance site](https://analytical-data-access.education.gov.uk/get-support/using-r-python-and-sql-in-databricks).
+
+The table below, taken from [NHS-R guidance](https://nhsdigital.github.io/rap-community-of-practice/implementing_RAP/notebooks_versus_ide_development/), shows a comparison of the features of notebooks and IDEs to help you decide where each would be most appropriately used:
```

Comment:
Could we move this table to after the warning box? Feels like we should be putting our steer on it first and then the extra information for them to understand in more depth afterwards?